tests/test_it.py

```diff
@@ -194,6 +194,28 @@ def test_I201_import_mock_config():
     ]
 
 
+def test_I201_most_specific_imports():
+    errors = run_flake8(
+        """
+        import urllib2
+        import urllib2.URLError
```

Comment:
Yeah I think `foo.bar` is preferred as it avoids confusion for future developers looking at this code